PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Core/Models/Appointment.cs

```diff
@@ -7,8 +7,9 @@ namespace PrivateHospitals.Core.Models;
 
 public class Appointment
 {
-    [Key]
+    [Key] 
     public int AppointmentId { get; set; }
+    public string? ExternalId { get; set; }
```

Comment:
This is not flexible approach. because if we Consume a few more companies, ids may still clash. Add CompanyId or SourceId to provide uniqueness